toucan_connectors/facebookads/facebookads_connector.py

```diff
@@ -0,0 +1,62 @@
+from typing import Dict
+from urllib.parse import urljoin
+
+import pandas as pd
+import requests
+from pydantic import Field
+
+from toucan_connectors.toucan_connector import ToucanConnector, ToucanDataSource
+
+from .helpers import ALLOWED_PARAMETERS_MAP, FacebookadsDataKind
```

Comment:
Meme point que plus haut sur les nommages

```suggestion
from .helpers import ALLOWED_PARAMETERS_MAP, FacebookAdsDataKind
```

Pour les noms de fichier : `facebook_ads/facebook_ads_connector.py`

Reply:
Renommé ici : https://github.com/ToucanToco/toucan-connectors/pull/349/commits/cd9695697a5154ed7d74ee70cc850cc473b1e026